test/dummy/db/migrate/20150717121532_create_articles.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.string :title
       t.text :body
       t.boolean :published, default: false
+      t.references :admin_user
```

Comment:
Added a reference which I use in the test as a custom orderable column. This doesn't have to be a reference but I thought it is a good idea to have a relation somewhere in the tests anyway :)